qiskit/aqua/operators/list_ops/summed_op.py

```diff
@@ -57,18 +58,59 @@ def distributive(self) -> bool:
         return True
 
     def add(self, other: OperatorBase) -> OperatorBase:
+        """ Return Operator addition of self and other, overloaded by ``+``.
+
+        Note:
+            This appends other to ``self.oplist`` without checking other is already included or not.
+            If you want to simplify this, please use ``SummedOp.simplify``.
```

Comment:
I will note that Sphinx has some markup to cross-reference classes, methods. modules etc. https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html Using these has the advantage that it will create a link that will take you directly to the class, method etc. so "..please use :meth:`simplify`" should work above. If you were referring to this from elsewhere you can make it fully qualified e.g.  "..please use :meth:`~qiskit.aqua.opertors.list_ops.SummedOp.simplify`" (the ~ means just print the last part, without that it would print the whole fully qualified list which maybe too much). I know I changed quite a few places to use this mechanism when I last went through docs to improve things like this. Just mentioning so you can bear in mind for future. If you search the docs you can find where this is done in other places if you want to look at examples.

Reply:
Thank you for details. I use :meth:`simplify`